conductor-clients/java/conductor-java-sdk/conductor-client/build.gradle

```diff
@@ -36,6 +35,7 @@ java {
 
 test {
     useJUnitPlatform()
+    maxParallelForks = 1
```

Comment:
@ystxn why is this required?

This might significantly slow down tests, since there will be no parallel test execution.

Reply:
I see now that tests weren't running because they were using JUnit4 annotations.

Reply:
this was to fix the non-determinism in this test. it strangely kept passing on my local but kept failing in github's build.
https://github.com/conductor-oss/conductor/actions/runs/12478325215/job/34825744258#step:4:72